README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
```

Comment:
lalala

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Readable, Secure, Inefficient, Inelegant, Reusable, Brittle